### PR TITLE
Adding Encointer & Robonomics networks

### PR DESCRIPTION
**Pull request by stepanLav**
# Encointer network
Screenshot 2022-01-28 at 11 47 01

# Robonomics network
Screenshot 2022-01-28 at 11 49 06

### CODE REVIEW

File: chains/v2/chains_dev.json [hunk 1]
```diff
@@ -1499,5 +1499,53 @@
...
+            "url": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/chains/v2/types/robonomics.json",
+            "overridesCommon": true
+        },
+        "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/chains/white/Robonomics.svg",
+        "addressPrefix": 32
+    },
+    {
+        "chainId": "7dd99936c1e9e6d1ce7d90eb6f33bea8393b4bf87677d675aa63c9cb3e8c5b5b",
+        "parentId": "b0a8d493285c2df73290dfb7e61f870f17b41801197a149ca93654499ea3dafe",
+        "name": "Encointer Launch",
```
Comment (antonkhvorov) [inline chains/v2/chains_dev.json:1530]: I think just "Encointer" should work fine

Comment (stepanLav) [reply chains/v2/chains_dev.json:1530]: Ok, it was exctract by script

File: chains/v2/types/encointer.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+    "runtime_id": 0,
```
Comment (valentunn) [inline chains/v2/types/encointer.json:2]: is it really 0?

File: chains/v2/chains_dev.json [hunk 3]
```diff
@@ -1499,5 +1499,53 @@
...
+                "url": "wss://api.kusama.encointer.org",
+                "name": "Encointer association node"
+            }
+        ],
+        "types": {
+            "url": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/chains/v2/types/encointer.json",
+            "overridesCommon": true
+        },
+        "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/chains/white/Encointer.svg",
+        "addressPrefix": 42
```
Comment (valentunn) [inline chains/v2/chains_dev.json:1549]: KSM token but generic address?

**Review by antonkhvorov — APPROVED** [review]

File: chains/v2/types/encointer.json (hunk 2 above)
Comment (stepanLav) [reply chains/v2/types/encointer.json:2]: Yeah, you can find it on screenshot

**Review by valentunn — APPROVED** [review]

**Pull request merged**